.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+    "printWidth": 100,
+    "tabWidth": 2,
+    "useTabs": false,
+    "semi": false,
+    "singleQuote": true,
+    "trailingComma": "none",
+    "bracketSpacing": true
+  }
+  
```

Comment:
there seems to be a space on this line, that's why you have the error `no new line at the end of file`.